docs/DEPLOY.md

```diff
@@ -0,0 +1,19 @@
+# Deploy
+
+##### `sls deploy`
+- Description: Deploy resource group containing function app and other resources. Also deploys Function App code
+- Two Approaches:
+  - Deploy resource group, upload packaged artifact directly to function app. Sets function app `RUN_FROM_PACKAGE` setting to `1`.
+  - Deploy resource group, upload packaged function app to blob storage with version name. Sets function app `RUN_FROM_PACKAGE` setting to path of zipped artifact in blob storage
```

Comment:
is this one blob storage per resource group?

Reply:
With the ARM template as is, there will be one blob storage account deployed per resource group. Seems like it's required for function apps. Don't know how hard that *required* is, but I've yet to see a function app deployed without a storage account deployed alongside it. @wbreza any insight here?